### ISSUE TITLE
Feature Request: Split by size

### ISSUE THREAD

**Issue by cs16lover** [AQUAOSOTech/tarsplitter#1]
First of all thank you for such a great open source project.

I think it will be nice to have a splitting feature by file sizes. 

For example; Split 1 TB tar file into 100 GB small tar files.

**Comment by ruffrey** [comment]
Thanks for the request! Is the goal to specify a "not larger than" size for each chunk? I could see it working like the following:

### Example:

Input: 250 mb file
Not larger than: 100 mb

### Option 1

Output:
- 1.tar - 100  mb
- 2.tar - 100 mb
- 3.tar - 50 mb

### Option 2

Output:
- 1.tar - 83.33  mb
- 2.tar -  83.33 mb
- 3.tar - 83.33  mb

Option 1 would be simplest to implement, I think.

**Comment by cs16lover** [comment]
Yes, exactly! "Not larger than" size option fits best. Because sometimes larger TAR files needs much more system resources to process on the fly.

**Comment by cs16lover** [comment]
Any updates?

**Comment by ruffrey** [comment]
Hey, we plan to do this, it is a useful suggestion, but likely not for another month.

**Comment by cs16lover** [comment]
Any updates?

**Comment by ruffrey** [comment]
Hi, thanks for bumping the thread. We don't have a timeline, but this is a useful feature we want to implement.
